reconcile/test/test_utils_gql.py

```diff
@@ -0,0 +1,72 @@
+import requests
+
+from reconcile.utils.gql import (
+    GqlApi,
+    GqlApiError,
+    GqlApiErrorForbiddenSchema,
+    GqlApiIntegrationNotFound,
+)
+from gql import Client
+from gql.transport.exceptions import TransportQueryError
+import pytest
+
+TEST_QUERY = """
+{
+    integrations: integrations_v1 {
+        name
+        description
+        schemas
+    }
+}
+"""
+
+
+def test_gqlapi_throws_gqlapierror_exception(mocker):
+    mocked_client = mocker.Mock(spec=Client)
```

Comment:
You pretty much always want `create_autospec` unless there's a compelling reason not to use it. I won't comment on each instance of this.